src/commonMain/kotlin/operations/logic/NotEquals.kt

```diff
@@ -0,0 +1,12 @@
+package operations.logic
+
+import operations.LogicOperation
+
+internal object NotEquals : LogicOperation, EqualsOperation {
+    override val key: String = "!="
+
+    override fun invoke(
+        expression: Any?,
+        data: Any?
+    ): Boolean = !safeCompare(expression) { first, second -> first == second }
```

Comment:
why you use negation instead of passing `first != second` operator?

Reply:
!=, !== is nothing more than negated ==/===. Also I put false as a default value if some nulls appear in comparison operations inside the ComparingOpration.kt. ComparingOperation.kt was mainly projected for less than, greater than operations. I decided to reuse it in equals operations with minimal adapting code needed. And here it is